vaccine_feed_ingest/runners/co/colorado_gov/parse.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+
+import json
+import pathlib
+import sys
+
+from lxml import etree
+from pykml import parser
+
+
+def parse_point(element):
+    coords = element.coordinates.text.strip().split(",")
+    return {"lat": coords[0], "long": coords[1]}
```

Comment:
From looking at this data, I _think_ these might be swapped. Latitude is -90 to 90, but this would put some latitudes in the -150s.